_includes/post-macros.html

```diff
@@ -675,118 +647,40 @@ <h1 class="h4">{{ icon }} {{ header }}</h1>
                 "From" date input
                 =================
             #}
-
-            {# No-JS date input #}
             {% set from_date = selected_filters_for_field('range_date_gte')[0] %}
-            <div class="form-l_col form-l_col-1-2 js-filter_range-date-wrapper">
+            {% set from_date_formatted = from_date | date('%m/%d/%Y') if from_date %}
+            <div class="form-l_col form-l_col-1-2">
                 <label for="{{ options.id_range_date_gte }}">
-                    From:
+                    From <span class='ie-date_format micro-desc'>(mm/dd/yyyy)</span>:
                 </label>
                 <input class="js-filter_range-date js-filter_range-date__gte"
```

Comment:
We would only need it if we are going to populate a hidden field via the onChange event. It is something I am thinking about implementing so we don't get a flash when we are formatting the date fields before submission.